.cirrus.yml

```diff
@@ -70,18 +94,19 @@ qa_task_filter_template: &QA_TASK_FILTER
     - build
   <<: *ONLY_IF_SONARSOURCE_QA
 
+# ----------------------------------------------
+# -------------------TASKS----------------------
+# ----------------------------------------------
+
 build_task:
   <<: *LINUX_3_5_CPU_7G
+  <<: *MAVEN_CACHE
   env:
-    SIGN_KEY: VAULT[development/kv/data/sign data.key]
-    PGP_PASSPHRASE: VAULT[development/kv/data/sign data.passphrase]
     # analysis on SonarCloud
-    SONAR_TOKEN: VAULT[development/kv/data/sonarcloud data.token]
     SONAR_HOST_URL: https://sonarcloud.io
```

Comment:
For me `SONAR_HOST_URL` should be moved next to `SONAR_TOKEN` to avoid a mismatch between token and URL. Now it will be hard to see the issue